### DISCUSSION TITLE
20.06.2021

### DISCUSSION THREAD

**Discussion by farag2** [farag2/Sophia-Script-for-Windows#207] [General]
# Before running the script read [Before running](https://github.com/farag2/Windows-10-Sophia-Script#fire-before-running-fire) & [How to use](https://github.com/farag2/Windows-10-Sophia-Script#how-to-use)

[![YT](https://img.youtube.com/vi/f529ucAipI8/0.jpg)](https://youtu.be/f529ucAipI8) [![YT](https://img.youtube.com/vi/MiQ85tVXQQA/0.jpg)](https://youtu.be/MiQ85tVXQQA)

***

[![ko-fi](https://www.ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/Q5Q51QUJC)

<a href="https://yoomoney.ru/to/4100116615568835"><img src="https://yoomoney.ru/i/shop/iomoney_logo_color_example.png" width=220px height=46px></a>

***

[![actions](https://img.shields.io/badge/Sophia%20News-Telegram-blue?style=flat&logo=Telegram)](https://t.me/SophiaNews) [![actions](https://img.shields.io/badge/Sophia%20Chat-Telegram-blue?style=flat&logo=Telegram)](https://t.me/Sophia_Chat) [![actions](https://img.shields.io/github/workflow/status/farag2/Windows-10-Sophia-Script/Build?label=GitHub%20Actions&logo=GitHub)](https://github.com/farag2/Windows-10-Sophia-Script/actions)

## Windows 10 2004/20H2/21H1 | LTSC

Diff from v5.10.7
[5.10.7...5.10.8](https://github.com/farag2/Windows-10-Sophia-Script/compare/5.10.7...5.10.8)

* The `UnpinTaskbarEdgeStore` function renamed into `UnpinTaskbarShortcuts`, and added feature to unpin the `Mail` shortcut
  * `UnpinTaskbarShortcuts -Shortcuts Edge, Store, Mail`.
* The `PowerManagementScheme` fucntion rename into `PowerPlan`;
* #195, #196, #200 closed;
* Updated the Chinese translation. Thanks to @flashercs;
* Minor changes. :feelsgood:
* The development focus shifted to [SophiApp](https://github.com/Sophia-Community/SophiApp) :rocket:

***

* Sophia Script Wrapper 2.0.2
  * Fixed runtime error;
  * Fixed PowerShell 7.1 launch (detects 5.1 or 7.1 and uses appropiate PowerShell to launch either);
  * Added online check for the latest Wrapper version. If you are using old version it will exit;
  * Added online check for the latest imported script version. If you are using old version it will disable directly running (run PowerShell button disabled);
  * UI changes.

|              File                      |      SHA-256   |
|:--------------------------------------:|:-----------------------------:|
| Sophia.Script.LTSC.v5.2.8.zip          |20769C2111DE11438C0F15BBDB71385A865765AA3C60C9AA36517D4A7D36A095|
| Sophia.Script.v5.10.8.PowerShell.7.zip |7B3D46CBEE668CEB27FBA5C15C5499548FD7880EA49C1C28D5C73A91EDDD2004|
| Sophia.Script.v5.10.8.zip              |F31A05648D351EC5C156A83B18E410B81BE96BDC45BA166E3672341B62A862B6|
| Sophia.Script.Wrapper.v2.0.2zip              |A2EA7BDE28C88645F7F3ADD2E2F18E4B45F754A1101C9AAFC2E479199B96E24B|

<hr /><em>This discussion was created from the release <a href='https://github.com/farag2/Windows-10-Sophia-Script/releases/tag/5.10.8'>20.06.2021</a>.</em>

**Comment by ghost** [comment]
Переключение на русский не работает

**Reply by farag2** [reply]
В каком смысле переключение на русский?
